chef/cookbooks/hyperv/recipes/do_ceilometer.rb

```diff
@@ -0,0 +1,8 @@
+raise if not node[:platform] == 'windows'
+
+include_recipe "hyperv::openstack_install_ceilometer"
+include_recipe "hyperv::openstack_install_neutron"
+include_recipe "hyperv::config_neutron"
+include_recipe "hyperv::config_ceilometer"
+include_recipe "hyperv::register_services_ceilometer"
+include_recipe "hyperv::register_services_neutron"
```

Comment:
I'm not sure if the order really matters, but usually neutron goes first. Wouldn't it be better to have also here ceilometer parts only after neutron parts?